### PR TITLE
add root service host_components list support

### PR DESCRIPTION
**Pull request by kozmagabor**
Could you merge it please?

### CODE REVIEW

**Comment by deathowl** [comment]
👍 lgtm

**Comment by jimbobhickville** [comment]
Crap, I didn't have notifications on so I had no idea there were PRs waiting for me.

File: ambariclient/models.py [hunk 1]
```diff
@@ -1219,10 +1219,25 @@ class View(base.QueryableModel):
...
+    fields = ('component_name', 'host_name', 'service_name')
 
 class RootServiceComponent(Component):
+    path = 'components'
     data_key = 'RootServiceComponents'
+    relationships = {
+        'host_components': RootServiceHostComponents,
+    }
 
+class RootServiceComponent(Component):
```
Comment (jimbobhickville) [inline ambariclient/models.py]: This appears to be duplicated from the class just above it.

Comment (kozmagabor) [inline ambariclient/models.py]: Yep, sorry, I fixed

**Comment by jimbobhickville** [comment]
0.5.10 released to pypi

**Pull request merged**